apps/betterangels-backend/notes/enums.py

```diff
@@ -75,6 +75,23 @@ class ServiceEnum(models.TextChoices):
     UNEMPLOYMENT_CERTIFICATION = "unemployment_certification", _("Unemployment Certification")
     VACCINE_PASSPORT = "vaccine_passport", _("Vaccine Passport")
     WATER = "water", _("Water")
+    BAG = "bag", _("Bag(s)")
```

Comment:
this looks good, just need to regenerate types. this is the diff I got when I ran `yarn nx affected -t validate-graphql-schema`
```py
diff --git a/apps/betterangels-backend/schema.graphql b/apps/betterangels-backend/schema.graphql
index 84e641f6..50b97eae 100644
--- a/apps/betterangels-backend/schema.graphql
+++ b/apps/betterangels-backend/schema.graphql
@@ -903,6 +903,23 @@ enum ServiceEnum {
   UNEMPLOYMENT_CERTIFICATION
   VACCINE_PASSPORT
   WATER
+  BAG
+  BATTERIES
+  BICYCLE_REPAIR
+  CLOTHES_SOCKS
+  CONSENT_TO_CONNECT
+  EBT
+  FEMININE_HYGIENE
+  FIRST_AID
+  HARM_REDUCTION_NARCAN
+  HARM_REDUCTION_SAFE_SMOKING
+  HARM_REDUCTION_TEST_KIT
+  HMIS_CONSENT
+  LAHOP
+  MEDI_CAL
+  SLEEPING_BAG
+  SSI_SSDI
+  TARP
   OTHER
 }
```

you can regenerate types with `ynx-generate`, which is an alias that combines `yarn nx affected -t generate-graphql-schema` and `yarn nx affected -t generate-graphql-types`. 

you can also run all the checks that run in CI with `ynx-precommit`. It'll show you anything you may have missed (except failing tests, those you'd run separately)